<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QReCheckDialog</class>
 <widget class="BaseDlg" name="QReCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QReCheckDialog</string>
  </property>
  <widget class="QTableView" name="assay_view">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>189</y>
     <width>620</width>
     <height>584</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView QScrollBar::handle:vertical  {
    margin: 46px 3px 3px 2px;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>800</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>800</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>620</width>
     <height>85</height>
    </rect>
   </property>
   <property name="bks" stdset="0">
    <string>gray</string>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>架号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rack_Edit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>位置号</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QPushButton" name="pos_1">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">1</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="buttons" stdset="0">
           <string>btn_recheck</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pos_2">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="buttons" stdset="0">
           <string>btn_recheck</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pos_3">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">3</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="buttons" stdset="0">
           <string>btn_recheck</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pos_4">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">4</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="buttons" stdset="0">
           <string>btn_recheck</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pos_5">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">5</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="buttons" stdset="0">
           <string>btn_recheck</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>189</y>
     <width>620</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>189</y>
     <width>1</width>
     <height>584</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>772</y>
     <width>620</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>189</y>
     <width>1</width>
     <height>584</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
